docs/installation/linux.md

```diff
@@ -52,3 +52,7 @@ There are more than one ways to setup PostgreSQL. For additional configuration,
 ### Possible error messages
 
 While installing, you might run into an error due to the pg gem requiring PostgreSQL libraries. If so, please run `sudo apt-get install libpq-dev` before retrying.
+
+While installing, you might run into an error due to the sass-rails gem requiring sassc, which requires the g++ compiler.  If so, please run `sudo apt-get install g++` before retrying.
```

Comment:
May be you want to wrap `sass-rails`, sassc and g++ inside single quote to highlight it then rest of words in the line.

Reply:
That's fair.  I was trying to match how it was done on line 54, but I agree that I like the quoted versions of package names better.  Would you like `pg` updated to match?